src/ui/czi-editor.css

```diff
@@ -67,6 +67,12 @@
     z-index: 1;
   }
 
+  .ProseMirror:not([data-layout]) {
+    min-height: 279mm;
+    padding-bottom: 15.5mm;
+    padding-top: 15.5mm;
+  }
+
```

Comment:
Please do verify that this doe snot break the embedded mode.
If the doc is embedded, it should not have a min-height.

Reply:
Verified that embedded mode doesn't have a `min-height` (the styling that does `min-height: unset` overrides the min-height in embedded mode)